ci-operator/step-registry/redhat-appstudio/rhtap-installer/e2e-test/redhat-appstudio-rhtap-installer-e2e-test-commands.sh

```diff
@@ -3,4 +3,32 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+export OPENSHIFT_PASSWORD OPENSHIFT_API RED_HAT_DEVELOPER_HUB_URL GITHUB_TOKEN \
+    GITHUB_ORGANIZATION QUAY_IMAGE_ORG APPLICATION_ROOT_NAMESPACE
+
 echo "start rhtap-installer e2e test"
+
+OPENSHIFT_PASSWORD="$(cat $KUBEADMIN_PASSWORD_FILE)"
+OPENSHIFT_API="$(yq e '.clusters[0].cluster.server' $KUBECONFIG)"
+timeout --foreground 5m bash  <<- "EOF"
+    while ! oc login "$OPENSHIFT_API" -u kubeadmin -p "$OPENSHIFT_PASSWORD" --insecure-skip-tls-verify=true; do
+        sleep 20
+    done
+EOF
+
+APPLICATION_ROOT_NAMESPACE="rhtap-e2e-ci"
+QUAY_IMAGE_ORG="rhtap_qe"
+GITHUB_ORGANIZATION="rhtap-rhdh-qe"
+GITHUB_TOKEN=$(cat /usr/local/rhtap-ci-secrets/rhtap/gihtub_token)
+RED_HAT_DEVELOPER_HUB_URL=https://"$(oc get route developer-hub -n rhtap -o jsonpath='{.spec.host}')"
+
+cd "$(mktemp -d)"
+
+oc create namespace openshift-gitops || true
+oc label namespace openshift-gitops argocd.argoproj.io/managed-by=openshift-gitops || true
```

Comment:
why does it need the above 2 commands?

Reply:
Damn, forgot to remove them. I will do it